sympy/solvers/solvers.py

```diff
@@ -350,7 +350,11 @@ def check_assumptions(expr, against=None, **assumptions):
         if test is expected:
             continue
         elif test is not None:
-            return False
+            if get is True:
+                return False, key
```

Comment:
In `(-5, integer=False, positive=True)` case `integer, positive` both assumptions are wrong. So both must be printed. For testcase : see how testcases are written in `solvers/tests/test_solvers.py ` for `check_assumptions`.